packages/contracts/src/contracts/current/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -126,32 +124,28 @@ contract MixinExchangeCore is
 
     /// @dev After calling, the order can not be filled anymore.
     ///      Throws if order is invalid or sender does not have permission to cancel.
-    /// @param order Order to cancel. Order must be Status.FILLABLE.
-    /// @return True if the order state changed to cancelled.
-    ///         False if the order was valid, but in an
-    ///               unfillable state (see LibStatus.STATUS for order states)
+    /// @param order Order to cancel. Order must be OrderStatus.FILLABLE.
     function cancelOrder(Order memory order)
         public
-        returns (bool)
     {
         // Fetch current order status
         OrderInfo memory orderInfo = getOrderInfo(order);
 
         // Validate context
-        assertValidCancel(order, orderInfo.orderStatus, orderInfo.orderHash);
+        assertValidCancel(order, orderInfo);
 
         // Perform cancel
-        return updateCancelledState(order, orderInfo.orderStatus, orderInfo.orderHash);
+        updateCancelledState(order, orderInfo.orderHash);
     }
 
     /// @dev Gets information about an order: status, hash, and amount filled.
     /// @param order Order to gather information on.
     /// @return OrderInfo Information about the order and its state.
-    ///                   See LibOrder.OrderInfo for a complete description.
+    ///         See LibOrder.OrderInfo for a complete description.
     function getOrderInfo(Order memory order)
         public
         view
-        returns (LibOrder.OrderInfo memory orderInfo)
+        returns (OrderInfo memory orderInfo)
```

Comment:
We seem to hop between `Order memory order` and `LibOrder.Order memory order`, what's the scenario where we want to include it versus reference it?

Reply:
We have a style meeting scheduled so I'll punt on the above till then.